src/Parser/RestoDruid/Modules/Features/StatWeights.js

```diff
@@ -0,0 +1,316 @@
+ * For now, I'm handling the most common big Druid stat buffs manually.
+ *
+ * Intellect -
+ * Math here is straightforward, as spells that scale with int scale directly and linearly with total int. Due to the 'all leather'
+ * bonus, each point of int gained is multiplied by 1.05, which has to be taken into account. The increase in power from
+ * one int is simply 1.05 / totalInt.
+ *
+ * Crit -
+ * The general formula is critChanceFromOneRating * healAmount, but we have to make sure to account for anything that makes a
+ * crit stronger than 2x (e.g. Drape of Shame), and we also have to account for if healAmount is already a crit. If the heal
+ * is already a crit then we double count the weight, so we first have to normalize it to the amount it would have done
+ * had it not crit. Special handling is needed for Living Seed obviously. We assume that higher crit will result in proportionally
+ * more Living Seeds, and so get the bonus to Living Seed as critChangeFromOneRating / critChance.
+ *
+ * Haste -
+ * Calculating the benefit to healing from being able to cast faster is a huge pain in the ass, particularly for Resto Druids
+ * who aren't very dependant on cast times. However, HoTs tick faster and effectively do increased healing per cast due to Haste.
+ * We make the decision to only calculate this benefit, which is fairly simple math:
+ * healAmount / (1 + hastePercentage) * hastePercentageFromOneRating
+ *
+ * Mastery -
+ * The mastery calculation is almost the same as Haste's, in that you normalize the heal amount to be the amount it would
+ * have been without any of the stat, and then multiply by the percentage gain from one rating. The one wrinkle is that mastery's
+ * strength is multiplied by the number of hots on target. Fortunately, we can easily calculate the number of HoTs on the target
+ * of each heal.
+ *
+ * Versatility -
+ * This is the same calculation as for haste, healAmount / (1 + versPercentage) * versPercentageFromOneRating.
+ * We don't currently consider the damage reduction bonus from vers, and it's probably not a good thing to add directly to
+ * the weight as damage reduction isn't quite the same thing as healing. It's easy enough to calculate just by skimming the
+ * appropriate percentage of each toPlayer_damage. I will probably add it to the tooltip in the future.
+ *
+ * Leech -
+ * Calculating this is different depending on if the player does or does not already have any Leech. If the player does have
+ * Leech we calculate this pretty much the same as int: for every Leech heal the increase in power from one leech is
+ * simply 1 / totalLeechRating. If the player does not already have Leech, things are much more difficult. I have not yet
+ * settled on an approach, and so this calculation is not yet implemented.
+ *
+ * Special Cases -
+ *
+ * A few special cases are considered. We're ignoring all Tranq healing, even ticks that don't overheal, under the assumption
```

Comment:
How about summing all Tranq healing per player and looking at the sum; if sum didn't include overhealing include it, otherwise ignore?

Reply:
That's exactly what I described when responding to blazyb above. It's what I plan to do.